src/main/resources/db/changelog/db.changelog-RDM-9052.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    
+    <changeSet id="RDM-9052-1" author="henry">
+	    <preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">SELECT count(c.relname) FROM pg_class c WHERE c.relkind = 'S' and c.relname = 'user_profile_audit_id_seq'</sqlCheck>
+	    </preConditions>
+		<addAutoIncrement columnDataType="int" columnName="id" tableName="user_profile_audit"/>  
```

Comment:
worth adding an explanatory <comment>?
We could for example say that from a certain version of liquibase all columns for which autoincrement is needed must be declared with` autoIncrement="true"` and this one was not. Therefore we need to add autoincrement with `addAutoIncrement`
(if my understanding of the problem is correct)

Reply:
done